pkg/webui/components/dropdown/attached.js

```diff
@@ -79,6 +88,20 @@ const AttachedDropdown = ({ attachedRef, onItemsClick, hover, onOutsideClick, ..
     [attachedRef, onOutsideClick],
   )
 
+  if (portalled) {
+    return (
+      <Portal visible={open} positionReference={attachedRef}>
```

Comment:
```suggestion
      <Portal positionReference={attachedRef}>
```
The portal can be rendered regardless.

Reply:
For the regular on click dropdown, maybe yes. But I found that if I do not use the visibility in the useEffect this won't work for the hover flyout tooltip in the sidebar. There I need to assign a different position for the dropdown portal on every hover, so that the position updates according to the item being hovered. If this is not dependent on visibility, the position of the dropdown will always be based on the last element in the sidebar.